packages/augur-tools/src/templates-template.ts

```diff
@@ -153,13 +153,65 @@ export enum TemplateInputType {
   USER_DESCRIPTION_DROPDOWN_OUTCOME = 'USER_DESCRIPTION_DROPDOWN_OUTCOME',
 }
 
+export interface ExtraInfoTemplateInput {
```

Comment:
This file is heavily duplicated in `packages/augur-artifacts/src/templates.ts`.

Reply:
it's the templates template file. it's used to generate the file in artifacts.

Reply:
ohhhh